UPGRADING

```diff
@@ -23,6 +23,11 @@ PHP 8.6 UPGRADE NOTES
 2. New Features
 ========================================
 
+- Intl:
+  . Added IntlNumberRangeFormatter class to format an interval of two numbers with a given skeleton, locale, IntlNumberRangeFormatter::COLLAPSE_AUTO, IntlNumberRangeFormatter::COLLAPSE_NONE, IntlNumberRangeFormatter::COLLAPSE_UNIT, IntlNumberRangeFormatter::COLLAPSE_ALL collapse and
```

Comment:
nit: each constant need to have their own line and short introduction[ like here](https://github.com/php/php-src/blob/8a1014128d22819a23bd278c2079a084fc1aef9a/UPGRADING#L229).

Reply:
Hmm.. I think those are constants, not class constants. I've followed the example we've used for IntlListFormatter from here: https://github.com/php/php-src/blob/8a1014128d22819a23bd278c2079a084fc1aef9a/UPGRADING#L269